src/env.ts

```diff
@@ -17,6 +17,10 @@ export interface Configuration {
   apiKey?: string;
   // Datadog API Key encrypted using KMS, only necessary when using metrics without log forwarding
   apiKMSKey?: string;
+  // Datadog API Key used for enabling monitor configuration through plugin
+  monitorsApiKey?: string;
```

Comment:
Why is this key different from the normal `apiKey`?

Reply:
I believe Tian and SF said that it would be better to keep the keys separate for security reasons

Reply:
I am still curious why this is needed -- maybe @tianchu or @sfirrin could shed light here.

It looks like for the normal `apiKey` we support KMS with the `apiKMSKey` option. But now we seem to require a second, separate API key just to use the monitors feature, and this key MUST be passed as plain text.